editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -2687,8 +2685,6 @@ void CanvasItemEditor::_draw_ruler_tool() {
 	if (tool != TOOL_RULER)
 		return;
 
-	bool is_snap_active = smart_snap_active ^ Input::get_singleton()->is_key_pressed(KEY_CONTROL);
```

Comment:
Won't this prevent using <kbd>Ctrl</kbd> as a modifier to temporarily toggle snapping?

Reply:
This is only for drawing the ruler units I guess. So I don't think it should hurt.

Reply:
I might be wrong, but I wonder since my last changes, if Ctrl has an impact on the grid snapping. It might be working only for the smart snapping now.

Reply:
Yeah, Ctrl doesn't affect grid snapping anymore, so I disabled it.